tools/devops/automation/templates/release/vs-insertion-prep.yml

```diff
@@ -68,6 +69,33 @@ stages:
         nuGetFeedType: external
         publishFeedCredentials: dnceng-dotnet6
 
+    - pwsh: |
+        mkdir $(Build.SourcesDirectory)/nugets-blob
+        cp $(Build.SourcesDirectory)/package/* $(Build.SourcesDirectory)/nugets-blob
+        cp $(Build.SourcesDirectory)/vs-msi-nugets/* $(Build.SourcesDirectory)/nugets-blob
+      displayName: "Copy content for the nugets blob."
+
+    - template: templates/common/upload-vs-insertion-artifacts.yml@sdk-insertions
+      parameters:
+        githubToken: $(GitHub.Token)
+        githubContext: $(NupkgCommitStatusName)
+        blobName: nuget-artifacts
```

Comment:
On the Android and MAUI side I'm also using ` $(NupkgCommitStatusName)` for the `blobName` parameter, should we do that here?

Reply:
+1

Reply:
```suggestion
        blobName: $(NupkgCommitStatusName)
```